.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,3 +9,5 @@
 ### Review Checks
 - [ ] All pages have automated accessibility checks via cypress
 - [ ] All `School` queries are correctly scoped - `eligible` when they need to be
+
+### How can I view this in a review app?
```

Comment:
Do we need to add a checklist, like: - [ ] Have you seeded the db with the right data etc

Reply:
It's a good question. I think it could be included in the review checks. I presume the developer writing out the instructions will have tried it in the preview app to see if they have the right data in place though.